shared-bindings/wifi/Radio.c

```diff
@@ -226,8 +226,8 @@ STATIC mp_obj_t wifi_radio_stop_station(mp_obj_t self) {
 MP_DEFINE_CONST_FUN_OBJ_1(wifi_radio_stop_station_obj, wifi_radio_stop_station);
 
 //|     def start_ap(self,
-//|                  ssid: ReadableBuffer,
-//|                  password: ReadableBuffer = b"",
+//|                  ssid: str,
+//|                  password: str = b"",
```

Comment:
The given default value is a `bytes`, not a `str`.
```suggestion
//|                  password: str = "",
```